sale_order_line_description/models/sale_order_line.py

```diff
@@ -7,9 +7,9 @@
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    @api.onchange("product_id")
-    def product_id_change(self):
-        res = super(SaleOrderLine, self).product_id_change()
+    @api.depends("product_id")
+    def _compute_name(self):
+        res = super()._compute_name()
         if not self.product_id:  # pragma: no cover
```

Comment:
You should change the function implementation to support multi recordset.

Reply:
done, thanks!